application/common/components/Emailer.php

```diff
@@ -190,16 +192,25 @@ public function getSiteStatus()
     {
         return $this->getEmailServiceClient()->getSiteStatus();
     }
-    
+
+    /**
+     * @param string $messageType
+     * @return null|string
+     */
     protected function getSubjectForMessage(string $messageType)
     {
         if ( ! empty($this->subjects[$messageType]) && strpos($this->subjects[$messageType], '%') !== false) {
             return sprintf($this->subjects[$messageType], $this->otherDataForEmails['idpDisplayName'] ?? '');
         }
         return $this->subjects[$messageType] ?? null;
     }
-    
-    protected function getViewForMessage(string $messageType, string $format)
+
+    /**
+     * @param string $messageType
+     * @param string $format
+     * @return string
+     */
+    protected function getViewForMessage(string $messageType, string $format): string
```

Comment:
I'm fine with having this PHPDoc block on this method, but what are your thoughts about what to do when the PHPDoc is only a duplication of the data in the method signature? I think I lean towards either removing it completely, or adding a description (of the purpose of the function) to the PHPDoc block. You're welcome to change it, remove it, or leave it as-is.

Reply:
I remember smelling that, but unfortunately did nothing about it. I'm definitely opposed to duplication of information, but something about leaving a DocBlock off entirely doesn't sit well with me. I'll add descriptive text so it adds value.